src/main/kotlin/platform/mcp/actions/CopyAtAction.kt

```diff
@@ -25,19 +25,33 @@ class CopyAtAction : SrgActionBase() {
     override fun withSrgTarget(parent: PsiElement, srgMap: McpSrgMap, e: AnActionEvent, data: ActionData) {
         when (parent) {
             is PsiField -> {
+                if (parent.containingClass == null) {
+                    return showBalloon("No SRG name found", e)
+                }
+                val classSrg = srgMap.getSrgClass(parent.containingClass!!) ?: return showBalloon(
```

Comment:
Is this `!!` necessary? Maybe you can remove it by extracting it to a local variable.

Reply:
Fixed